com.microsoft.mrtk.spatialmanipulation/BoundsControl/BoundsControl.cs

```diff
@@ -734,7 +734,7 @@ internal void OnHandleSelectEntered(BoundsHandleInteractable handle, SelectEnter
         private static readonly ProfilerMarker TransformTargetPerfMarker =
             new ProfilerMarker("[MRTK] BoundsControl.TransformTarget");
 
-        private void TransformTarget()
+        protected virtual void TransformTarget()
```

Comment:
I'm a little worried about exposing this function.  Can you please explain what behavior you're trying to override.   Perhaps there's a cleaner way to do this instead of exposing this function.

Reply:
Hi @AMollis,

We want to implement custom rotation, scaling, translation methods in different use cases